interceptor.go

```diff
@@ -129,6 +145,10 @@ func (i *interceptor) start(ctx context.Context) error {
 		}
 	}(ctx)
 
+	// We consider ourselves connected now.
+	disconnectedNodesGaugeMetric.Dec()
+	defer disconnectedNodesGaugeMetric.Inc()
```

Comment:
An alert does need to have some delay period, because we do periodic stream disconnects.

Reply:
Anyway, in prom/datadog, we use time sample (if lnmux_disconnected_nodes > 1 for 5 min then alert)